File: app/src/main/java/habibkazemi/ir/lingualyrics_android/ui/savedlyrics/SavedLyricsFragment.java
package habibkazemi.ir.lingualyrics_android.ui.savedlyrics;


import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import habibkazemi.ir.lingualyrics_android.R;

/**
 * A simple {@link Fragment} subclass.
 */
public class SavedLyricsFragment extends Fragment {


    public SavedLyricsFragment() {
        // Required empty public constructor
    }


    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_saved_lyrics, container, false);
    }

}

File: app/src/main/java/ir/habibkazemi/lingualyrics/ui/savedlyrics/SavedLyricsFragment.kt
package ir.habibkazemi.lingualyrics.ui.savedlyrics


import android.os.Bundle
import androidx.fragment.app.Fragment
import android.view.LayoutInflater
import android.view.Menu
import android.view.MenuInflater
import android.view.View
import android.view.ViewGroup

import ir.habibkazemi.lingualyrics.R


/**
 * A simple [Fragment] subclass.
 */
class SavedLyricsFragment : Fragment() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setHasOptionsMenu(true)
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_saved_lyrics, container, false)
    }


    override fun onCreateOptionsMenu(menu: Menu?, inflater: MenuInflater?) {
        super.onCreateOptionsMenu(menu, inflater)
        menu?.findItem(R.id.search)?.isVisible = false
    }

}
